kqueen/utils/loggers.py

```diff
@@ -0,0 +1,24 @@
+import os
+import yaml
+import logging.config
+import logging
+
+
+def setup_logging(default_path='/code/kqueen_ui/utils/logger_config.yml', default_level=logging.INFO, env_key='LOG_CFG'):
```

Comment:
Remove hardcoded default path or put them to reasonable path.

Reply:
Please don't use `LOG_CFG`, at least prefix it with `KQUEEN_`.

Or better use configuration via `kqueen.config`.

Reply:
ok, got it